dfinity/oisy-wallet
# feat(frontend): auto-enable ERC20 tokens when converting from ckERC20

# Motivation

When the user converts ckERC20 to ERC20, it is useful that the ERC20 token is enabled if it is not. We use the same flow as per the Receive button: when the user click the Convert button if the token is not enabled, it will be.
Thanks to @peterpeterparker for the suggestion.

# Tests

In local replica, the following test worked:

- Disable Sepolia USDC.
- Convert ckSepoliaUSD to Sepolia USDC.
- Check if Sepolia USDC was enabled.


https://github.com/user-attachments/assets/7a3a5ea3-f91f-4182-b181-7481350fb545
